core/templates/components/common-layout-directives/navigation-bars/side-navigation-bar.component.html

```diff
@@ -109,88 +182,110 @@
 
   <ul>
     <li [ngClass]="{'active': currentUrl === '/learn/math'}">
-      <a (click)="toggleClassroomSubmenu()" class="protractor-mobile-test-classroom">
-        <i class="material-icons oppia-sidebar-menu-icon">school</i>
-        <span>{{ 'I18N_SIDEBAR_CLASSROOM' | translate }}</span>
-      </a>
-      <div [hidden]="!classroomSubmenuIsShown" class="classroom-submenu">
+      <div (click)="togglelearnSubmenu()" class="oppia-sidebar-expandalbe-button protractor-mobile-test-learn" >
+        <i class="material-icons oppia-sidebar-menu-icon expand-icon oppia-sidebar-menu-icon-transition"
+          [ngClass]="{'oppia-sidebar-menu-icon-transition': learnSubmenuIsShown}">expand_more
+        </i>
+        <span class="oppia-sidebar-submenu-toggle text-uppercase" [ngClass]="{'oppia-sidebar-submenu-toggle': learnSubmenuIsShown}">
+              {{ 'I18N_SIDEBAR_LEARN' | translate }}
+        </span>
+      </div>
+      <div [hidden]="!learnSubmenuIsShown" class="oppia-sidebar-submenu-upperborder">
+        <div class="oppia-submenu-image-position">
+          <picture>
+            <source type="image/webp"
+                    [srcset]="this.getStaticImageUrl('/sidebar/learn-math-logo.webp')">
+            <source type="image/png"
+                    [srcset]="this.getStaticImageUrl('/sidebar/learn-math-logo.png')">
+            <img [src]="getStaticImageUrl('/sidebar/learn-math-logo.png')"
+                 class="oppia-submenu-image"
+                 alt="learn-math-logo">
+          </picture>
+        </div>
         <a href="/learn/math" routerLink="/learn/math" class="protractor-mobile-test-mathematics-link">
-          <span>{{ 'I18N_SIDEBAR_CLASSROOM_BASIC_MATHS' | translate }}</span>
+          <span>{{ 'I18N_SIDEBAR_MATH_FOUNDATIONS' | translate }}</span>
+          <P class="oppia-sidebar-learn-subtext">{{ 'I18N_SIDEBAR_MATH_FOUNDATIONS_DESCRIPTION' | translate }}</P>
         </a>
+        <div class="submenu-notop-margin">
+          <a href="/learn/math/place-values" routerLink="/learn/math/place-values">
+            <span>{{ 'I18N_SIDEBAR_SUBMENU_PLACE_VALUES' | translate }}</span>
+          </a>
+        </div>
+        <div class="submenu-notop-margin">
+          <a href="/learn/math/addition-subtraction" routerLink="/learn/math/addition-subtraction">
+            <span>{{ 'I18N_SIDEBAR_SUBMENU_ADDITION_AND_SUBTRACTION' | translate }}</span>
+          </a>
+        </div>
+        <div class="submenu-notop-margin">
+          <a href="/learn/math/multiplication" routerLink="/learn/math/multiplication">
+            <span>{{ 'I18N_SIDEBAR_SUBMENU_MULTIPLICATION' | translate }}</span>
+          </a>
```

Comment:
These are the links to topics in the classroom. We should check for them to be present in the respective classroom and then display the names of the first three accordingly rather than directly adding in the html.

Reply:
done!